This is probably a problem caused by compiling with Cython 2 instead
of Cython 3 (which we don't have in aports yet).

--- a/psycopg_c/psycopg_c/_psycopg/adapt.pyx
+++ b/psycopg_c/psycopg_c/_psycopg/adapt.pyx
@@ -14,6 +14,7 @@
 # Copyright (C) 2020 The Psycopg Team
 
 from typing import Any
+from psycopg.abc import AdaptContext, Buffer  # XXX-Patched: line added
 
 cimport cython
 
--- a/psycopg_c/psycopg_c/_psycopg/generators.pyx
+++ b/psycopg_c/psycopg_c/_psycopg/generators.pyx
@@ -7,6 +7,8 @@
 from cpython.object cimport PyObject_CallFunctionObjArgs
 
 from typing import List
+from psycopg.abc import PQGenConn  # XXX-Patched: line added
+from psycopg.pq.abc import PGconn, PGresult  # XXX-Patched: line added
 
 from psycopg import errors as e
 from psycopg.pq import abc, error_message
--- a/psycopg_c/psycopg_c/pq/pgconn.pyx
+++ b/psycopg_c/psycopg_c/pq/pgconn.pyx
@@ -17,6 +17,8 @@
     """
     pid_t getpid()
 
+from typing import Optional, List, Sequence, Tuple  # XXX-Patched: line added
+
 from libc.stdio cimport fdopen
 from cpython.mem cimport PyMem_Malloc, PyMem_Free
 from cpython.bytes cimport PyBytes_AsString
